presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -242,7 +243,7 @@ protected AggregationNode build()
 
     public ApplyNode apply(Assignments subqueryAssignments, List<Symbol> correlation, PlanNode input, PlanNode subquery)
     {
-        return new ApplyNode(idAllocator.getNextId(), input, subquery, subqueryAssignments, correlation);
+        return new ApplyNode(idAllocator.getNextId(), input, subquery, subqueryAssignments, correlation, new NullLiteral());
```

Comment:
I think this `ApplyNode` wasn't created out of null literal. It was created out of we-don't-know-what. `Optional`? A comment (eg something like below)?
```
Node originSubquery = new NullLiteral(); // doesn't matter for tests
```